### PR TITLE
DBZ-2578 Update Vitess documentation for adding primary key support

### PR DESCRIPTION
**Pull request by keweishang**
https://issues.redhat.com/browse/DBZ-2578

Requires https://github.com/debezium/debezium-connector-vitess/pull/7

### CODE REVIEW

**Comment by keweishang** [comment]
@rgibaiev

File: documentation/modules/ROOT/pages/connectors/vitess.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 toc::[]
 
-{prodname}'s Vitess connector captures row-level changes in the shards of a Vitess link:https://vitess.io/docs/concepts/keyspace/[keyspace]. Vitess versions 7 and 8 are supported.
+{prodname}'s Vitess connector captures row-level changes in the shards of a Vitess link:https://vitess.io/docs/concepts/keyspace/[keyspace]. Vitess version 8 is supported.
```
Comment (Naros) [inline documentation/modules/ROOT/pages/connectors/vitess.adoc:15]: I assume by this change this implies that Vitess 7 is no longer supported?  I ask just so we can make the appropriate adjustments at https://debezium.io/releases/.

Comment (keweishang) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:15]: Hi @Naros, thanks for letting me know about https://debezium.io/releases/

Correct. We currently only focus on supporting the latest Vitess GA version, which is 8 at the moment. All integration tests are tested against Vitess 8. The reason being that the Vitess API is still changing rapidly. At some point, when Vitess API is becoming more stable, we can start considering supporting multiple Vitess versions.

Comment (gunnarmorling) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:15]: Makes sense. In particular as this connector still is incubating at this point; once it has matured, we'll need to consider compatibility for existing users, too.

Comment (keweishang) [reply documentation/modules/ROOT/pages/connectors/vitess.adoc:15]: I definitely agree :)

**Review by gunnarmorling — APPROVED** [review]
LGTM; will merge once https://github.com/debezium/debezium-connector-vitess/pull/7 is in.

**Comment by gunnarmorling** [comment]
Thx!

**Pull request merged**